app.js

```diff
@@ -0,0 +1,6 @@
+// The date and time is display to check connectivity with the html file.
+var dateToday = new Date();
+var date = dateToday.getDate() + "/" + (dateToday.getMonth()+1) + "/" + dateToday.getFullYear();
```

Comment:
Since you already starting some js code I can recommend to take a look at [string literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)
So code could be 
```js
var date = `${dateToday.getDate()}/${(dateToday.getMonth()+1}/${dateToday.getFullYear()}`;
```

Reply:
great, thanks, I'll change it :)